SSE isn't available on any PPC platform, not just Darwin.

Donnie Berkholz <dberkholz@gentoo.org>

SSE isn't available on any platform but the ones supporting SSE

Luca Barbato <lu_zero@gentoo.org>

--- mlt-0.2.2.orig/src/modules/motion_est/filter_motion_est.c	2006-08-21 16:55:50.000000000 -0700
+++ mlt-0.2.2/src/modules/motion_est/filter_motion_est.c	2006-08-21 16:55:26.000000000 -0700
@@ -34,7 +34,7 @@
 #include <sys/time.h>
 #include <unistd.h>
 
-#ifndef __DARWIN__
+#ifdef __SSE__
 #include "sad_sse.h"
 #endif
 
@@ -612,7 +612,7 @@
 	 } /* End column loop */
 	} /* End row loop */
 
-#ifndef __DARWIN__
+#ifdef __SSE__
 	asm volatile ( "emms" );
 #endif
 
@@ -656,7 +656,7 @@
 static void init_optimizations( struct motion_est_context_s *c )
 {
 	switch(c->mb_w){
-#ifndef __DARWIN__
+#ifdef __SSE__
 		case 4:  if(c->mb_h == 4)	c->compare_optimized = sad_sse_422_luma_4x4;
 			 else				c->compare_optimized = sad_sse_422_luma_4w;
 			 break;
